grafana-plugin/src/components/CheatSheet/CheatSheet.config.ts

```diff
@@ -172,3 +178,138 @@ export const slackMessageTemplateCheatSheet: CheatSheetInterface = {
     },
   ],
 };
+
+export const alertGroupDynamicLabelCheatSheet: CheatSheetInterface = {
+  name: 'Dynamic Label cheatsheet',
+  description: 'Dynamic Label template is used to extract value for a specified key from the alert payload.',
```

Comment:
How is it visible in the UI? Does it make sense to duplicate the name? Maybe:
```
Dynamic Label template is used to extract value for a specified key from the alert payload.
``` 

->

```
Used to extract dynamic value from the alert payload.
```

Reply:
It's looks like this: I think It's ok to duplicate, since name is a cheetsheet's name, not a template name
<img width="632" alt="image" src="https://github.com/grafana/oncall/assets/20221722/4b4046bb-a08b-491c-a2c5-ac61d21ed686">